### DISCUSSION TITLE
4G USB MODEM HUAWEI 3276s

### DISCUSSION THREAD

**Discussion by FabrizioBocci** [Ysurac/openmptcprouter#3355] [General]
Hello,
i was wondering if there is a way to use Huawei 3276s USB modems as WAN connection.
If so , how should i configure the port?
thanks in advance for your contribution

**Comment by FabrizioBocci** [comment]
Somehow i figured out the way to configure an USB modem. 
After many attempts to make it work, i had to manually specify the APN which usually is taken automatically. Now i have a message  GATEWAY DOWN and i don't know how to solve it.

**Reply by Ysurac** [reply]
I have same problem but only with some ISP.
I'm not sure why for now...

**Reply by Ysurac** [reply]
What is the Network used ?
I have the problem with Orange in France and Luxemburg

**Reply by FabrizioBocci** [reply]
I'm using Windtre in Italy

**Comment by FabrizioBocci** [comment]
I have installed :
openmptcprouter-v0.61-snapshot-6.6-r0+26018-84a48ce400-x86-64-generic-squashfs-combined-efi.img
on an N100 box, do you have any idea why i don't find BBR2 ?

**Reply by Ysurac** [reply]
Because it use BBR3 only named BBR.
